[Hyperdrive] MySQL (#21346)
cloudflare/cloudflare-docs: src/content/docs/hyperdrive/reference/supported-databases-and-features.mdx

---
pcx_content_type: concept
title: Supported databases and features
sidebar:
  order: 2
---

## Database support

> Details on which database engines and/or specific database providers are supported are detailed in the following table.

Reviewer: ```suggestion
The following table shows which database engines and/or specific database providers are supported.
```
